app/controllers/games_controller.rb

```diff
@@ -10,6 +11,7 @@ def new
 
   def create
     @game = Game.create(game_params)
+    @game.add_player
```

Comment:
I'm not familiar with this syntax. Does this work the same as `@game.players.create`? Or does an `#add_player` method need to be added to the game model? Also, the associations need adjusting. I actually updated them in my branch before I saw this pull. I think if we deal with the small merge conflict between yours and mine the player creation should work properly.